app/config/index.js

```diff
@@ -159,10 +159,14 @@ const config = convict({
 });
 
 // Load environment dependent configuration
-const env = config.get('NODE_ENV') || 'development';
+const namespace = config.get("OPENSHIFT_APP_NAMESPACE") || 'dev-local';
+const chunks = namespace.split('-');
+const env = config.get('NODE_ENV') !== 'development' 
+  ? chunks[chunks.length-1] || 'local' 
+  : 'local';
 
 try {
-  config.loadFile('./' + env + '.json');
+  config.loadFile('./config/' + env + '.json');
 }
 catch(e){
   console.log(e);
```

Comment:
Noticing that this will start the app anyway if we don't find the json file. Is this what we want?

Reply:
Yes, that was original code